### PR TITLE
perf: use vite replace craco

### PR DESCRIPTION
**Pull request by HuberTRoy**
[SQN-1727](https://onfinality.atlassian.net/browse/SQN-1727)

## Desc

- Remove craco(Vite replace)/Sentry(don't need it)/Husky(Use rush replace)/Lint-staged(Use rush replace)/web-vitals(don't need it, devtools can do the same things).
- Use Vite(dev) + tsc(prod) + Rollup(prod) replace craco.
   1. tsc for potential type errors(no ts errors now).
   2. Build time from 6~9min reduce to 2min.
   3. Hot update for dev.
- Fix some bugs after upgrade.

### Build

SCR-20230801-omdn

The following tests based on dev & prod.

### Hot update

SCR-20230801-ojvp

### Project relatives.

[x] Add/Start/Update/Remove/Stop.
[x] Announce status.
[x] Flex Plan.
[x] Logs/Fetch informations...

SCR-20230801-omaf

### Networks

SCR-20230801-ozid

### Account

[x] Update metadata.
[x] Add new controller and set active and remove.

SCR-20230801-ozol

SCR-20230801-ozrq

SCR-20230801-padd

SCR-20230801-papb

### CODE REVIEW

File: apps/indexer-admin/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 SKIP_PREFLIGHT_CHECK=true
-REACT_APP_IPFS_GATEWAY=https://unauthipfs.subquery.network/ipfs/api/v0
-REACT_APP_QUERY_REGISTRY_PROJECT=https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai
-REACT_APP_GQL_PROXY=https://gql-proxy.subquery.network
+VITE_APP_IPFS_GATEWAY=https://unauthipfs.subquery.network/ipfs/api/v0
+VITE_APP_QUERY_REGISTRY_PROJECT=https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai
```
Comment (mzxyz) [inline apps/indexer-admin/.env:3]: Move this one to `env.development` and replace with Kepler one

File: apps/indexer-admin/src/pages/project-details/types.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export interface AsyncMemoReturn<T> extends AsyncData<T> {
 
 export type ProjectConfig = {
   baseConfig: {
-    networkEndpoint: string;
+    networkEndpoints: string;
```
Comment (mzxyz) [inline apps/indexer-admin/src/pages/project-details/types.ts:83]: [question] should be `string[]`

Comment (HuberTRoy) [reply apps/indexer-admin/src/pages/project-details/types.ts:83]: @mzxyz oo, right.

**Review by mzxyz — APPROVED** [review]

**Comment by mzxyz** [comment]
Nice work!  LGTM

**Pull request merged**